src/main/java/domain/numberscontainer/Ticket.java

```diff
@@ -1,21 +1,31 @@
 package domain.numberscontainer;
 
+import java.util.Set;
 import java.util.stream.Collectors;
 
-public class Ticket extends SixLottoNumbers {
+public class Ticket {
+    private static final String TICKET_DELIMITER = ", ";
+    private static final String START_MARK = "[";
+    private static final String END_MARK = "]";
 
-    public Ticket(SixLottoNumbersDTO sixLottoNumbersDTO) {
-        super(sixLottoNumbersDTO);
+    private LottoNumbers lottoNumbers;
+
+    public Ticket(Set<LottoNumber> lottoNumbers) {
+        this.lottoNumbers = new LottoNumbers(lottoNumbers);
+    }
+
+    public Ticket(String lottoNumbers) {
+        this.lottoNumbers = new LottoNumbers(lottoNumbers);
     }
 
     public boolean contains(LottoNumber number) {
-        return this.sixLottoNumbers.contains(number);
+        return this.lottoNumbers.contains(number);
     }
 
     @Override
     public String toString() {
-        return this.sixLottoNumbers.stream()
+        return this.lottoNumbers.get().stream()
```

Comment:
```suggestion
        return this.lottoNumbers.stream()
```

일급 컬렉션의 경우 `stream()` 메서드를 생성해주면 스트림 사용이 편합니다!